Development/nmos/ocsp_behaviour.h

```diff
@@ -0,0 +1,25 @@
+#ifndef NMOS_OCSP_BEHAVIOUR_H
+#define NMOS_OCSP_BEHAVIOUR_H
+
+#include <functional>
+#include "nmos/certificate_handlers.h"
+
+namespace slog
+{
+    class base_gate;
+}
+
+namespace nmos
+{
+    struct model;
+
+    namespace experimental
+    {
+        struct ocsp_state;
+    }
+
+    // callbacks from this function are called with the model locked, and may read or write directly to the model
```

Comment:
is this comment accurate? when are these handlers called?

Reply:
You're saying all other calls of these two handlers are "protected" by a read lock on the model need? I wonder if that's consistent/necessary/good idea or whether would be more natural to call with it unlocked?

In any case, I suspect the content is wrong about writes being OK...
Ah, I also just spotted an inconsistent space before `&` on next line...

Reply:
Good point, I was just trying to keep it consistent, but is it really necessary, maybe not. Then we will have to check all other threads, behaviour is the same too or not!

> Ah, I also just spotted an inconsistent space before & on next line...

where?